tests/test_z_cmdline.py

```diff
@@ -45,41 +44,6 @@ def wait(self):
     report.expect("logpopen")
 
 
-def test__resolve_pkg(tmpdir, mocksession):
```

Comment:
This file at the moment is more or less a general collection of everything else. We'll slowly phase it out. For now I've just moved out tests related to the version parsing, that have nothing to do with the command line.